### PR TITLE
Fixing spelling.

### PR DESCRIPTION
**Pull request by mahela97**
File - https://github.com/mahela97/Terasology/blob/2aa5cbbf6b08bdf8826acc3b3f8517815df9eb5c/engine/src/main/java/org/terasology/engine/utilities/procedural/SimplexNoise.java

Line -73

### Contains

Fix spelling errors in the documentation. The correct name of the variable is TILEABLE1DMAGICNUMBER.

- Before it was  ->
@see SimplexNoise#tileable1DMagicNumber

- Now ->
@see SimplexNoise#TILEABLE1DMAGICNUMBER

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -12,5 +12,5 @@
     </writeAnnotations>
   </component>
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="11" project-jdk-type="JavaSDK" />
-</project>
```
Comment (pollend) [inline .idea/misc.xml:16]: can you revert this change.

**Review by pollend — CHANGES_REQUESTED** [review]

File: .idea/misc.xml (hunk 1 above)
Comment (mahela97) [reply .idea/misc.xml]: sure
